Generation/src/PythiaInterface.h

```diff
@@ -22,14 +27,19 @@ class PythiaInterface: public GaudiAlgorithm {
   virtual StatusCode finalize();
 
 private:
+
   // Pythia8 engine
   Pythia8::Pythia * m_pythia;
-  /// the name of the Pythia parameter input file
+
+  // Name of Pythia configuration input file
```

Comment:
Why did you change the doxygen comment to a normal comment? Please re-add the third "/".